src/botPage/view/blockly/blocks/tools/webhook.js

```diff
@@ -0,0 +1,168 @@
+/* eslint-disable no-underscore-dangle */
+import { translate } from '../../../../../common/i18n';
+import { expectValue } from '../shared';
+
+Blockly.Blocks.webhook = {
+    init() {
+        this.jsonInit({
+            message0: translate('Webhook URL: %1'),
+            args0   : [
+                {
+                    type: 'input_value',
+                    name: 'WEBHOOK_URL',
+                },
+            ],
+            colour           : '#dedede',
+            previousStatement: null,
+            nextStatement    : null,
+            tooltip          : translate('Sends a POST request to a URL'),
+        });
+
+        this.itemCount_ = 3;
```

Comment:
Perhaps we could start with `1` item so the user doesn't feel that they need to add 3 right away.